addons/grenadeLauncherAmmo/config.cpp

```diff
@@ -6,7 +6,7 @@ class CfgPatches {
 		name = QUOTE(ADDON);
 		url = "$STR_grad_URL";
 		requiredVersion = 1.0;
-		requiredAddons[] = {"grad_main", "A3_Data_F", "A3_Weapons_F"};
+		requiredAddons[] = {"grad_main", "A3_Data_F", "A3_Weapons_F", "rhsusf_c_weapons"};
```

Comment:
you're patching their classes, but if they dont exist, you'll just simply create a few near-empty classes without erroring out. So I guess we dont need to set this dependency?

Reply:
oh wait, this is used for load order, right? so if the dependency is not here, RHS might load later than… nvm :crying_cat_face:

Reply:
It is used for load order but it also shows a warning if the addon is missing, I could move the RHS stuff into a separate pbo (`grenadeLauncherAmmoRhs`?) if the dependency bothers you.

Reply:
i've been looking into all the other modules in gruppe_adler_mod, and the only places with non-ACE/CBA dependencies are 
* uh1y (depends on RHS) and
* flashlight (depends on asdg_jointrails)

idk. this module brings value even when we play without RHS, otoh it's been a looong time since we did that, so I guess it's ok. ^^

opinions @McDiod @AdlerSalbei @nomisum @Synch2k @DerZade ?

Reply:
Well we have the GM Mod upcoming whitch will lead to an extra repo without RHS.

ASDG_Jointrails should be in CBA.

I think an optional PBO would be perfect.

Reply:
I removed the RHS dependency in grad_equipment not long ago, so I guess I naturally have to vote for everything that removes RHS dependency. I believe an optional PBO would be the best solution.

Reply:
I'm with @Synch2k. Dependencies in an extra PBO would be nice.

Reply:
Alright, I moved the RHS stuff into `grenadeLauncherAmmoRhs`.